src/renderer/components/+workloads-deployments/deployments.tsx

```diff
@@ -8,21 +8,16 @@ import "./deployments.scss";
 import React from "react";
 import { observer } from "mobx-react";
 import type { RouteComponentProps } from "react-router";
-import { Deployment, deploymentApi } from "../../../common/k8s-api/endpoints";
-import type { KubeObjectMenuProps } from "../kube-object-menu";
-import { MenuItem } from "../menu";
-import { Icon } from "../icon";
-import { DeploymentScaleDialog } from "./deployment-scale-dialog";
-import { ConfirmDialog } from "../confirm-dialog";
+import type { Deployment } from "../../../common/k8s-api/endpoints";
 import { deploymentStore } from "./deployments.store";
 import { eventStore } from "../+events/event.store";
 import { KubeObjectListLayout } from "../kube-object-list-layout";
 import { cssNames } from "../../utils";
 import kebabCase from "lodash/kebabCase";
 import orderBy from "lodash/orderBy";
 import { KubeObjectStatusIcon } from "../kube-object-status-icon";
-import { Notifications } from "../notifications";
 import type { DeploymentsRouteParams } from "../../../common/routes";
+import { DeploymentMenu } from "./deployment-menu";
```

Comment:
Note: This component was extracted to separate file to avoid circular dependency